packages/user-menu/src/UserNavigationUrls.js

```diff
@@ -0,0 +1,87 @@
+const notificationsUrl = (env) => {
```

Comment:
in een aparte file
export const NOTIFICATION_URLS = {
  dev: 'http://'
  acc: 'http://',
  prod: 'http://'
  local: 'http../
}

export const APROFIEL_URLS = {}

// apart bestand
notificationsUrl = env => NOTIFICATION_URLS[env] || NOTIFICATION_URLS.prod
}